frontend/src/components/PluginContextProvider.tsx

```diff
@@ -69,16 +98,24 @@ export function PluginContextProvider(props: { children?: ReactNode }) {
         ocpApi,
       }}
     >
-      <div style={{ position: 'relative', height: '100%', width: '100%' }}>
-        <div style={{ position: 'absolute', height: '100%', width: '100%' }}>
-          <AcmToastProvider>
-            <AcmToastGroup />
-            <AcmTablePaginationContextProvider localStorageKey="clusters">
-              {props.children}
-            </AcmTablePaginationContextProvider>
-          </AcmToastProvider>
+      <AcmFeedbackModalProvider>
```

Comment:
What is the purpose of this context provider? I do not see any consumers for the context, and removing it seemed to have no effect.

Reply:
removing